src/validateRegisterForm.test.js

```diff
@@ -26,5 +26,60 @@ describe(`Function 'validateRegisterForm':`, () => {
     expect(invalidPassword.message).toBe('Password is invalid.');
   });
 
-  // write more tests here
+it('should input the valid password, which includes at least 8 characters', () => {
+  const validPassword = validateRegisterForm('123olesia');
+  expect(validPassword.code).to.be.equal(200);
+  expect(validPassword.message).to.be.equal('Password is valid.');
+  });
+
+  it(`should return error for too long password`, () => {
+    const invalidPassword = validateRegisterForm('test@mail.com', 'Pass1@12jjdjdjdkkckcclclcjgkkcndcldc');
```

Comment:
long password is 17 chars (boundary value)
for short - 7
in this task you should include all possible combinations of password (as per the test design techniques) and all possible combinations of email 
add more tests